README.md

```diff
@@ -26,9 +26,15 @@ The plug-in currently supports the following distributions:
 * Ubuntu 18.04 and 20.04 LTS
 * CentOS 7 and 8
 
-It requires [Libfabric](http://github.com/ofiwg/libfabric/). Please see the
+It requires [Libfabric](http://github.com/ofiwg/libfabric/),
+[NCCL](https://github.com/NVIDIA/nccl),
+[HWLOC](https://www.open-mpi.org/projects/hwloc/), and (if you want to
+run tests) an MPI Implementation. Please see the
 [Release notes](http://github.com/aws/aws-ofi-nccl/releases) for
-information on version compatibility.
+information on version compatibility.  We recommend using the
+distribution version of hwloc, which can be installed with `yum
+install hwloc-devel` on many RPM based distributions and `apt install
+libhwloc-dev` on many DPKG based distibutions.
```

Comment:
Does the latest version of the plugin just require the `hwloc-devel`/`libhwloc-dev` package, or also the `hwloc-plugins`/`libhwloc-plugins` package?

Reply:
It does not require the plugins package.  Just hwloc.